### DISCUSSION TITLE
一つのキーを押すと他のキーも反応してしまう

### DISCUSSION THREAD

**Discussion by g2112026** [Yowkees/keyball#261] [Q&A] [answered]
キーが反応しなかったので、キーソケットを付け直したところそのキーを押すと他のキーも一緒に反応してしまいます。
どのような原因が考えられるのでしょうか？
例：「；」のキーを押しているのに少し離れた「j」のキーが同時に反応して「；j」と入力される

**Comment by Yowkees** [comment]
機種はKeyball61でしょうか？
そうですと、;とjは２倍マトリクスで対になっているキーですので、どちらかのダイオードの向きが間違っている可能性があります。
またダイオードが破損している可能性もありますので、新しいものに交換してみてください。
お手数ですがご確認お願いいたします。

**Reply by g2112026** [reply]
ご回答ありがとうございます。
機種はKeyball61になります。
ダイオードの向きは合っているようなので、;とjの周辺のダイオードをいくつか付け替えてみたのですが、改善されませんでした。
具体的にどこのダイオードか見当つきますでしょうか？

**Comment by Yowkees** [comment, marked as answer]
赤丸の２つになります。
ご確認お願いします。
![KB61R_KEYfix](https://github.com/Yowkees/keyball/assets/84209772/9b6fa786-69d5-4571-b20a-e9837b02ed4e)

**Reply by g2112026** [reply]
ご回答ありがとうございます。
画像の左下のダイオードを付け替えましたら、改善されました。
これで完成させることができると思います！
丁寧にサポートしていただけてとても助かりました。

**Comment by Yowkees** [comment]
無事に修理できたようでよかったです。安心しました。
